### PR TITLE
[CI:DOCS] Cirrus: Fix version-check to only run on `main` job

### PR DESCRIPTION
**Pull request by cevich**
A conditional in `version-check` bypasses the test for PRs.  However, it appears it was intended to execute during the daily cirrus-cron runs. However, the cron-job it references (`nightly`) doesn't exist.  This is causing the test to run for every merge into `main`, and never run for `main` branch cirrus-cron job.  Fix the name so the test **ONLY** runs for the `main` branch cron-job.

Also, [since the test is currently failing](https://cirrus-ci.com/task/6183669669298176), update the docs as per the output instructions.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by cevich** [comment]
Ugh...my logicing is bad...fixing...

**Comment by cevich** [comment]
...okay, I think that's correct now.

**Comment by edsantiago** [comment]
LGTM, sorry for not getting this right

**Comment by cevich** [comment]
NP.  Now we can be properly annoyed by it firing-off every week, yay! :grin:

File: docs/version-check
Comment (edsantiago) [inline docs/version-check:32]: Uh, no, this breaks the "run me interactively" part.

Too much churn, so I won't offer a suggestion. Once this stabilizes, I can take a new look.

**Review by edsantiago — CHANGES_REQUESTED** [review]

File: docs/version-check
Comment (cevich) [reply docs/version-check:32]: Sorry I don't follow.  My assumption is the script should only run in full when triggered by cirrus-cron for the `main` job (i.e. `$CIRRUS_CRON=="main"` - the branch and job name happen to be the same) and never any other time.  Is there some other condition where it should also operate in some way?

Comment (cevich) [reply docs/version-check:32]: OIC, you mean when run by a human.  Gotcha, I can fix that.

**Comment by mheon** [comment]
LGTM

File: docs/version-check
Comment (edsantiago) [inline docs/version-check:32]: The `-z` clause is not really needed, and pretty confusing; I'd really like to see it gone.

Comment (cevich) [reply docs/version-check:32]: right.

**Comment by cevich** [comment]
Force-push: Addressed Ed's concern.

**Comment by edsantiago** [comment]
Why repush before #17541 merges?

**Comment by cevich** [comment]
> Why repush before https://github.com/containers/podman/pull/17541 merges?

So I have a PR to test when cirrus fixes the issue :grin:

**Review by edsantiago — APPROVED** [review]

**Review by vrothberg — APPROVED** [review]
/hold cancel

**Pull request merged**
